packages/component-base/src/virtualizer-iron-list-adapter.js

```diff
@@ -45,6 +45,8 @@ export class IronListAdapter {
       this.scrollTarget.style.overflow = 'auto';
     }
 
+    console.log(getComputedStyle(this.scrollContainer).position);
+
```

Comment:
Oops?

Reply:
It was for Tatu to check the output in his broken environment. However, it didn't help. I'll remove it.

Reply:
Check out https://github.com/vaadin/web-components/issues/5590#issuecomment-1467744426 and https://github.com/vaadin/web-components/issues/5590#issuecomment-1470143851